test/hermes_test/test_cli.py

```diff
@@ -102,8 +102,36 @@ def test_haggis_with_post():
     assert not result.exception
 
 
+def test_haggis_with_path():
+    runner = CliRunner()
+    result = runner.invoke(cli.haggis, args=('--path', './'))
+
+    assert not result.exception
```

Comment:
A more explicit assertion would be nice. Cannot surely state what, though... maybe `assert result.exception is None`

Reply:
Not sure how this works in Python, but isn't `assert result.exception is None` the same as `assert not result.exception`?

Reply:
Technically: no.

`assert not result.exception` is coerced to `assert not bool(result.exception)` which might have some unforseen effects. E.g., if you have something like `throw ''` (i.e., you throw an empty string instead of a proper exception, which is allowed) you'd end up with `not result.exception` evaluating to `True` while `result.exception is not None` evaluating to `False`.

Also, some might think that the proposed version would make a better PyTest backtrace (but this is speculative).

Reply:
Re-thinking this (and looking at `git blame`) the `not result.exception` looks really great... 😇  However, more explicit might be still be a good idea (but I won't deny my approval for this).